### PR TITLE
Master

### PR DESCRIPTION
**Pull request by goodman-capu**
紧急修复 本地测试完成

### CODE REVIEW

File: api/client.php [hunk 1]
```diff
@@ -311,15 +311,21 @@ function seemain() {
...
-        $statement="select * from capubbs.mainpage where id=-3";
+        $statement="select * from capubbs.mainpage where id=-3";//iOS客户端检测更新方法已改进 不需要这个了
         $results=mysql_query($statement,$con);
         $results=mysql_fetch_row($results);
         echo '<iostext><![CDATA['.$results[2].']]></iostext>';
         echo '<iosurl><![CDATA['.$results[3].']]></iosurl>';
         echo '<iosversion><![CDATA['.$results[4].']]></iosversion>';
         echo '</info>'."\n";
-
+
```
Comment (huxuan) [inline api/client.php]: Please remove this redundant indent.

File: api/client.php [hunk 2]
```diff
@@ -311,15 +311,21 @@ function seemain() {
...
         $results=mysql_query($statement,$con);
         $results=mysql_fetch_row($results);
         echo '<iostext><![CDATA['.$results[2].']]></iostext>';
         echo '<iosurl><![CDATA['.$results[3].']]></iosurl>';
         echo '<iosversion><![CDATA['.$results[4].']]></iosversion>';
         echo '</info>'."\n";
-
+        
         $statement="select * from capubbs.mainpage where id=1 order by number desc limit 0,10";//增加了首页最多显示的通知数量
+        $moreinfo=@$_REQUEST['more'];//兼容老版本无法显示超过六条通知的Bug
```
Comment (huxuan) [inline api/client.php]: So this is where cause the force quit? Just for confirmation.

File: api/jiekouapi.php [hunk 3]
```diff
@@ -319,7 +319,7 @@ function register($con,$ip) {
 			exit;
 		}
 
-        //$code=mysql_real_escape_string(@$_REQUEST['username']);//取消验证码制度
+        //$code=mysql_real_escape_string(@$_REQUEST['code']);//取消验证码制度
```
Comment (huxuan) [inline api/jiekouapi.php]: Why do you need to change this?
I prefer to leave as it is before.

File: api/client.php (hunk 2 above)
Comment (goodman-capu) [inline api/client.php]: 是的 我拿老版本测试了 而且我很早就发现了这个bug

File: api/jiekouapi.php (hunk 3 above)
Comment (goodman-capu) [inline api/jiekouapi.php]: 额 之前那个是被改过的 原版是我现在提交的这个。。。 已经被注释掉了 是个测试

Comment (goodman-capu) [inline api/jiekouapi.php]: 原版应该是‘code’ 那个‘username’是我取消验证码制度时测试用的 现在这样才叫‘leave as it is before’ 23333

Comment (huxuan) [inline api/jiekouapi.php]: 好吧呀，那这个就先这样吧~

**Comment by huxuan** [comment]
Thnaks for your job.
A small suggestion is please using meaningful message for COMMIT and PULL REQUEST.

**Pull request merged**